alma/controllers/admin/AdminAlmaCategories.php

```diff
@@ -328,6 +324,23 @@ public function renderView()
         return $this->renderList();
     }
 
+    /**
+     * AdminController::renderList() override.
+     *
+     * @see AdminController::renderList()
+     */
+    public function renderList()
+    {
+        $this->addRowAction('voidaction');
+
+        return parent::renderList();
+    }
+
+    public function displayVoidactionLink($token = null, $id)
```

Comment:
**Major**: switch `$token` and `$id` arguments (an argument with default value cannot be set before a required argument)

minor: write `displayVoidActionLink` instead of `displayVoidactionLink` (if possible)

Reply:
I fix it :) thanks